tests/test_density_GaussianDensity.py

```diff
@@ -103,6 +103,38 @@ def test_sum_3d(self):
         # This has discretization error as well as single-precision error
         assert np.isclose(np.sum(gd.density), 1, atol=1e-4)
 
+    def test_sum_val_2d(self):
+        # Ensure that the Gaussian convolution sums to average of the values
+        width = 100
+        r_max = 49
+        sigma = 10
+        num_points = 1
+        box_size = width
+        box, points = freud.data.make_random_system(
```

Comment:
```suggestion
        system = freud.data.make_random_system(
```

Since you're just handing the system around without using the box or points independently, you can just pass the tuple around.